### ISSUE TITLE
Request to configure on restart 

### ISSUE THREAD

**Issue by denbue** [MacPass/MacPassHTTP#21]
Each time I restart MacPass I have to reconnect chromeIPass again with MacPassHTTP. I am not sure if this is an issue with MacPassHTTP or chromeIPass. On Windows and official KeePass it works fine.

`chromeIPass has not been configured. Press the connect button to register and pair with KeePassHttp.`

**Comment by mstarke** [comment]
I've done the following:
1. Lauch MacPass
2. Unlock Database
3. Launch Chrome
4. Pair Chrome with MacPass DB
5. Restart MacPass
6. Open DB

and always get "chromeIPass has been configured using the identifier <identifier> and is successfully connected to KeePassHttp."

**Comment by mstarke** [comment]
If I go on a website that has a form while MacPass is closed, ChromeIPass complains about not finding any KeePassHTTP server. But after opening MacPass and unlocking and reloading the page, everything is fine. What might be necessary

**Comment by denbue** [comment]
I checked everything twice and found the error, but I am not sure if this is an actual issue. If I connect chromeIPass and MacPass the credentials are written in the database. Before I close MacPass I have to manually save the database because MacPass closes without any prompt, which results in a prompt to reconnect on startup.

**Comment by mstarke** [comment]
That's an actual regression bug I introduced with MacPass 0.6 and hope to fix with 0.7 and history support on entries, as this caused the stripping of undo/redo functionality in KeePassKit.

TL;DR; known bug, will be fixed shortly :)
